packages/theming/src/palette/index.js

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
Would it be possible to combine all the palettes into this file? When the contents of the file are less than the license header it might be too spread out 😄 

Also, you can use:

```js
import azure from './azure'; 
```

for your imports